client/src/components/QuestionsAndAnswers/QuestionListItem/QuestionListItem.jsx

```diff
@@ -3,14 +3,56 @@ import './QuestionListItem.scss';
 import QuestionListItemAnswer from '../QuestionListItemAnswer/QuestionListItemAnswer.jsx';
 import QuestionListItemQuestion from '../QuestionListItemQuestion/QuestionListItemQuestion.jsx';
 
-const QuestionsListItem = () => {
+const QuestionsListItem = ({
+  id,
+  author,
+  question,
+  date,
+  helpfulness,
+  reported,
+  answers,
+  onQuestionHelpulButtonClick,
+  onAnswerHelpulButtonClick,
+  setShowAddAnswerModal
+}) => {
+  console.log('Question List Item question', question);
+  console.log('Question List Item answers', answers);
   return (
-    <li>
-      <QuestionListItemQuestion />
-      <ul>
-        <QuestionListItemAnswer />
-        <QuestionListItemAnswer />
-      </ul>
+    <li className="questions-list-item">
+      <div className="question-wrapper">
+        <h3>Q:</h3>
+        <QuestionListItemQuestion
+          author={author}
+          question={question}
+          date={date}
+          helpfulness={helpfulness}
+          id={id}
+          reported={reported}
+          onHelpulButtonClick={onQuestionHelpulButtonClick}
+          setShowAddAnswerModal={setShowAddAnswerModal}
+        />
+      </div>
+      { !!Object.keys(answers).length && <div className="answer-wrapper">
+        <h3>A:</h3>
+        <ul className="answer-list">
+          {Object.keys(answers).map(answerId => {
+            const answer = answers[answerId];
+
+            return (
+              <QuestionListItemAnswer
+                key={answer.id}
```

Comment:
Passing in the entire answer object might reduce the number of lines in the code. Just a suggestion for minor improvement.

Reply:
same as above